src/main/res/values/strings.xml

```diff
@@ -803,8 +803,8 @@
     <string name="pref_imap_folder_warn_disable_defaults">If you change this option, make sure, your server and your other clients are configured accordingly.\n\nOtherwise things may not work at all.</string>
     <!-- deprecated -->
     <string name="pref_watch_sent_folder">Watch Sent Folder</string>
-    <string name="pref_send_copy_to_self">Send Copy to Self</string>
-    <string name="pref_send_copy_to_self_explain">Required when using this account on multiple devices.</string>
+    <string name="pref_send_copy_to_self">Multi-device Mode</string>
```

Comment:
This string is difficult to translate, "Multi-" prefix does not have a clear meaning. "Mode" sounds like if it is something that you can turn on and off such as "Airplane mode" or "Do not disturb mode" to get additional capabilities or disable some features.

I'd call it "Multiple devices" instead.

Reply:
the translations don't have to be literal, about just "multiple devices" I am not sure, let's wait for r10s if he has some suggestion or prefers "Multiple devices", anyways the setting has a subtitle explaining when it needs to be enabled

Reply:
we are speaking of "multi-device" [already in the help](https://delta.chat/en/help), so that seems fine. 

but i would still add a translator hint, explaining what it is and that translations do not need to be literal